modules/platforms/dotnet/Apache.Ignite.Core.Tests/Cache/Query/CacheDmlQueriesTestSchema.cs

```diff
@@ -0,0 +1,259 @@
+        {
+            Sql("CREATE TABLE " + SCHEMA_NAME_1 + '.' + TBL_NAME + " (s1_key INT PRIMARY KEY, s1_val INT)");
+            Sql("CREATE TABLE " + SCHEMA_NAME_2 + '.' + TBL_NAME + " (s2_key INT PRIMARY KEY, s2_val INT)");
+            Sql("CREATE TABLE " + Q_SCHEMA_NAME_3 + '.' + TBL_NAME + " (s3_key INT PRIMARY KEY, s3_val INT)");
+            Sql("CREATE TABLE " + SCHEMA_NAME_4 + '.' + TBL_NAME + " (s4_key INT PRIMARY KEY, s4_val INT)");
+
+            Sql("INSERT INTO " + SCHEMA_NAME_1 + '.' + TBL_NAME + " (s1_key, s1_val) VALUES (1, 2)");
+            Sql("INSERT INTO " + SCHEMA_NAME_2 + '.' + TBL_NAME + " (s2_key, s2_val) VALUES (1, 2)");
+            Sql("INSERT INTO " + Q_SCHEMA_NAME_3 + '.' + TBL_NAME + " (s3_key, s3_val) VALUES (1, 2)");
+            Sql("INSERT INTO " + SCHEMA_NAME_4 + '.' + TBL_NAME + " (s4_key, s4_val) VALUES (1, 2)");
+
+            Sql("UPDATE " + SCHEMA_NAME_1 + '.' + TBL_NAME + " SET s1_val = 5");
+            Sql("UPDATE " + SCHEMA_NAME_2 + '.' + TBL_NAME + " SET s2_val = 5");
+            Sql("UPDATE " + Q_SCHEMA_NAME_3 + '.' + TBL_NAME + " SET s3_val = 5");
+            Sql("UPDATE " + SCHEMA_NAME_4 + '.' + TBL_NAME + " SET s4_val = 5");
+
+            Sql("DELETE FROM " + SCHEMA_NAME_1 + '.' + TBL_NAME);
+            Sql("DELETE FROM " + SCHEMA_NAME_2 + '.' + TBL_NAME);
+            Sql("DELETE FROM " + Q_SCHEMA_NAME_3 + '.' + TBL_NAME);
+            Sql("DELETE FROM " + SCHEMA_NAME_4 + '.' + TBL_NAME);
+
+            Sql("CREATE INDEX t1_idx_1 ON " + SCHEMA_NAME_1 + '.' + TBL_NAME + "(s1_val)");
+            Sql("CREATE INDEX t1_idx_1 ON " + SCHEMA_NAME_2 + '.' + TBL_NAME + "(s2_val)");
+            Sql("CREATE INDEX t1_idx_1 ON " + Q_SCHEMA_NAME_3 + '.' + TBL_NAME + "(s3_val)");
+            Sql("CREATE INDEX t1_idx_1 ON " + SCHEMA_NAME_4 + '.' + TBL_NAME + "(s4_val)");
+
+            Sql("SELECT * FROM " + SCHEMA_NAME_1 + '.' + TBL_NAME);
+            Sql("SELECT * FROM " + SCHEMA_NAME_2 + '.' + TBL_NAME);
+            Sql("SELECT * FROM " + Q_SCHEMA_NAME_3 + '.' + TBL_NAME);
+            Sql("SELECT * FROM " + SCHEMA_NAME_4 + '.' + TBL_NAME);
+
+            Sql("SELECT * FROM " + SCHEMA_NAME_1 + '.' + TBL_NAME
+                + " JOIN " + SCHEMA_NAME_2 + '.' + TBL_NAME
+                + " JOIN " + Q_SCHEMA_NAME_3 + '.' + TBL_NAME
+                + " JOIN " + SCHEMA_NAME_4 + '.' + TBL_NAME);
+
+            VerifyTables();
+
+            Sql("DROP TABLE " + SCHEMA_NAME_1 + '.' + TBL_NAME);
+            Sql("DROP TABLE " + SCHEMA_NAME_2 + '.' + TBL_NAME);
+            Sql("DROP TABLE " + Q_SCHEMA_NAME_3 + '.' + TBL_NAME);
+            Sql("DROP TABLE " + SCHEMA_NAME_4 + '.' + TBL_NAME);
+        }
+
+        /// <summary>
+        /// Verify tables.
+        /// </summary>
+        private void VerifyTables()
+        {
+            Sql("SELECT SCHEMA_NAME, KEY_ALIAS FROM SYS.TABLES ORDER BY SCHEMA_NAME", res => {
+                res.Equals(new List<List<object>> {
```

Comment:
Here and below - `Equals` does not do what you expect:
* It is a reference comparison, will be always `false` here
* It returns `bool`, which is just ignored, so the test does not do anything

Instead, we should say `Assert.AreEqual(new List..., res)` (first arg is `expected`, second `actual`).